### PR TITLE
Updated Japanese translation

### PR DESCRIPTION
**Pull request by tkusano**
Updated Japanese translation

### CODE REVIEW

**Comment by tkusano** [comment]
@Hima-Zinn @dolciss @oboenikui @noritada @middlingphys @hibiki @reindex-ot @haoyayoi @vyv03354

Could someone review this?

note:
I translated "Clip 🐴 clop 🐴" as "クリップ 🐴 クロップ 🐴", but it could be translated as "パカラッ 🐴 パカラッ 🐴" or something like that.

**Review by Hima-Zinn — COMMENTED** [review]
After a quick check, it seems to be fine.
Need to get a review from others.

File: src/locale/locales/ja/messages.po [hunk 1]
```diff
@@ -857,7 +857,7 @@ msgstr "気象"
 
 #: src/components/dms/ChatEmptyPill.tsx:39
 msgid "Clip 🐴 clop 🐴"
-msgstr ""
+msgstr "クリップ 🐴 クロップ 🐴"
```
Comment (Hima-Zinn) [inline src/locale/locales/ja/messages.po:860]:
```suggestion
msgstr "パカラッ 🐴 パカラッ 🐴"
```

こっちの方が馴染み深いのではないかという指摘がありました。
https://bsky.app/profile/mimonelu.net/post/3ksxvbmtvcs23

**Review by Hima-Zinn — COMMENTED** [review]
Please Check a review.

File: src/locale/locales/ja/messages.po (hunk 1 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po:860]: Created a poll
https://bsky.app/profile/did:plc:urj4pkche2yvpr5ilj4ykteu/post/3ksxwnz6vwz25

Comment (Hima-Zinn) [reply src/locale/locales/ja/messages.po:860]: thanks.

Comment (tkusano) [reply src/locale/locales/ja/messages.po]: I follow the will of the people
https://poll.blue/p/GGev6n/0
image

**Comment by tkusano** [comment]
| changes | source |
|----------|-------|
| https://github.com/bluesky-social/social-app/pull/4144/commits/a5816f7d78ecd7f60ecf86350db6159403aa5801 | #4145 |
| https://github.com/bluesky-social/social-app/pull/4144/commits/a8c8eba24ea2e1e766be0f8188f48fa478b143e1 | https://poll.blue/p/GGev6n?v=0 |
| https://github.com/bluesky-social/social-app/pull/4144/commits/3722addc381b579040f3538ffb1d0cc639d4500b | #4146 |

**Comment by hibiki** [comment]
I checked, I don't think it's a problem!

**Comment by tkusano** [comment]
https://github.com/bluesky-social/social-app/pull/4144/commits/26a42798c5e6b8a3cce8d0c34437abc4bcb6cf63 ref. https://github.com/bluesky-social/social-app/commit/0e77280310e03482525b31d5a907a2820509641b

**Comment by tkusano** [comment]
Translated new strings added by https://github.com/bluesky-social/social-app/commit/d2c42cf16905a8904dcfbba4825ca5f8abc3f253 and https://github.com/bluesky-social/social-app/commit/f7ee532a8521b96afd2cb358980d9b25bd59c29a

**Review by pfrazee — APPROVED** [review]
Thanks all!!

**Pull request merged**